conformance/apis/v1alpha1/profilereport.go

```diff
@@ -23,26 +23,28 @@ type ProfileReport struct {
 	// (e.g. "AdminNetworkPolicy", "BaselineAdminNetworkPolicy")
 	Name string `json:"name"`
 
-	// Core indicates the core support level which includes the set of tests
+	// Standard indicates the standard support level which includes the set of tests
 	// which are the minimum the implementation must pass to be considered at
 	// all conformant.
-	Core Status `json:"core"`
+	Standard Status `json:"standard"`
 
-	// Extended indicates the extended support level which includes additional
+	// Experimental indicates the experimental support level which includes additional
 	// optional features which the implementation may choose to implement
-	// support for, but are not required.
-	Extended *ExtendedStatus `json:"extended,omitempty"`
+	// support for, but are unstable, ungraduated to standard features. However if
```

Comment:
"Optional" isn't a good word to describe Experimental.

Experimental indicates the experimental support level, which includes additional features
that have not yet graduated to Standard.

Reply:
done